balancer/rls/internal/picker_test.go

```diff
@@ -0,0 +1,671 @@
+// +build go1.10
+
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package rls
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc/balancer"
+	"google.golang.org/grpc/balancer/rls/internal/cache"
+	"google.golang.org/grpc/balancer/rls/internal/keys"
+	rlspb "google.golang.org/grpc/balancer/rls/internal/proto/grpc_lookup_v1"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	defaultTestTimeout = 1 * time.Second
+	maxAge             = 5 * time.Second
+)
+
+func initKeyBuilderMap(t *testing.T) keys.BuilderMap {
+	t.Helper()
```

Comment:
These are all const values in here- it's more appropriate IMO to fatal with the line number of the MakeBuilderMap call instead.